util-images/network/netperfbenchmark/pkg/worker/worker.go

```diff
@@ -92,10 +93,10 @@ var (
 	// -e : enhanced reports, gives more metrics for udp.
 	// -s : run in server mode.
 	// -P numOfClients: handle <numOfClients> number of clients before disconnecting.
-	udpClientArguments = []string{"-c", "{serverPodIP}", "-u", "-f", "K", "-l", "20", "-b", "1M", "-e", "-i", "1", "-t", "{duration}"}
+	udpClientArguments = []string{"-c", "{serverPodIP}", "-u", "-f", "K", "-e", "-i", "1", "-t", "{duration}"}
 	udpServerArguments = []string{"-s", "-f", "K", "-u", "-e", "-i", "{duration}", "-P", "{numberOfClients}"}
 	tcpServerArguments = []string{"-s", "-f", "K", "-i", "{duration}", "-P", "{numberOfClients}"}
-	tcpClientArguments = []string{"-c", "{serverPodIP}", "-f", "K", "-l", "20", "-b", "1M", "-i", "1", "-t", "{duration}"}
+	tcpClientArguments = []string{"-c", "{serverPodIP}", "-f", "K", "-i", "1", "-t", "{duration}"}
```

Comment:
I see that you removed here few flags and added "-b 750M" in yaml config. What do you think about splitting this change into two PRs(or just commits)? One PR containing logic for parsing flags, moving the one that you want to change to yaml file and second PR modifying flags? So first PR will be no-op and second will modify behaviour.

Reply:
Sure,will raise 2 seperate PR's.

Reply:
I have created https://github.com/kubernetes/perf-tests/pull/1853/ for yaml changes.